src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -57,17 +57,24 @@ case class BaggedSingleResult(
   override def getExpected(): Seq[Double] = Seq(expected)
 
   private lazy val expected = treePredictions.sum / treePredictions.length
-  private lazy val treeVariance: Double = treePredictions.map(x => Math.pow(x - expected, 2.0)).sum / treePredictions.length
+  private lazy val treeVariance: Double = {
+    assert(treePredictions.length > 1, "Bootstrap variance undefined for fewer than 2 bootstrap samples." + treePredictions.length)
+    treePredictions.map(x => Math.pow(x - expected, 2.0)).sum / treePredictions.length
+  }
 
-  override def getStdDevObs(): Option[Seq[Double]] = Some(Seq(treeVariance))
+  override def getStdDevObs(): Option[Seq[Double]] = Some(Seq(scalarUncertainty))
 
-  override def getStdDevMean(): Option[Seq[Double]] = Some(Seq(scalarUncertainty))
+  override def getStdDevMean(): Option[Seq[Double]] = Some(Seq(Math.sqrt(treeVariance)))
```

Comment:
My benchmarking suggested that `Math.sqrt(treeVariance) * rescale` would be `getStdDevObs` not `getStdDevMean`.  Do we know that it goes to zero as the number of samples of a noisy signal increases?

Reply:
I have observed that it goes to zero in the number of *duplicated* samples of a noisy signal (i.e. observations with the same inputs) but things get harder to analyze with more complicated base learners. We'd still be able to analyze desired behavior of a linear base learner, but that's not much different from GuessTheMean.